src/components/backend-ai-data-view.ts

```diff
@@ -976,7 +980,7 @@ export default class BackendAIData extends BackendAIPage {
       this.allowedGroups = group_info.groups;
     }
     this.cloneFolderNameInput.value = await this._checkFolderNameAlreadyExists(
-      this.cloneFolderName,
+      this.cloneFolderID,
```

Comment:
`_checkFolderNameAlreadyExists`: it seems to be unused now. However, this function needs a name instead of an ID because it compares names.